examples/fungible-token/src/lib.rs

```diff
@@ -5,15 +5,15 @@
 *  - JSON calls should pass U128 as a base-10 string. E.g. "100".
 *  - The contract optimizes the inner trie structure by hashing account IDs. It will prevent some
 *    abuse of deep tries. Shouldn't be an issue, once NEAR clients implement full hashing of keys.
-*  - The contract tracks storage difference before and after the call. If the storage increases,
+*  - The contract tracks the change in storage before and after the call. If the storage increases,
 *    the contract requires the caller of the contract to attach enough deposit to the function call
 *    to cover the storage stake difference.
```

Comment:
```suggestion
*    to cover the storage cost.
```

Reply:
If the storage decreases, the contract will issue a refund for the cost of the released storage.